src/fmu/dataio/_objectdata_provider.py

```diff
@@ -699,6 +704,9 @@ def _get_columns(self) -> list[str]:
             columns = list(self.obj.columns)
         else:
             logger.debug("arrow")
+            from pyarrow import Table
```

Comment:
I wonder if and when these lazy imports start to become numerous enough to bite us later on... the import doesn't seem excessively slow to my mind, about half a second (at least as far as Python is concerned). Probably few enough to be OK now, but will there be more?

Reply:
Ive wondered so as well. But i think for now, its fine. But if i go back and look at older versions it looks like there was a period of time where we could not really upon pyarrow being installed.